app/scripts/robot/gamepad.coffee

```diff
@@ -26,6 +31,7 @@ angular.module('daemon.gamepad', [])
           callCallbacksOnce()
           # update the timestamps
           _currentTimestamps[i] = gamepad.timestamp
+          sendGamepadData(JSON.stringify(gamepad));
```

Comment:
@AllanYangZhou doesn't this just send every joystick once?

Also it's probably a good idea if we include debouncing: http://socket.io/docs/ see volatile